### DISCUSSION TITLE
SSH access to pi after installing Anthias

### DISCUSSION THREAD

**Discussion by Grougrou4** [Screenly/Anthias#1862] [General]
I have read lot of topics about it but nothing work. So my problem is how to have ssh access to my pi after anthias install ?

**Comment by vpetersson** [comment]
If you want SSH access, you need to do a manual installation on Raspbian/Raspberry Pi OS.

**Comment by Grougrou4** [comment]
do i need docker to perform the install or not ?

**Reply by vpetersson** [reply]
Yes, docker is a requirement for installing as you can see in the installation script.

**Comment by Grougrou4** [comment]
is docker installing itself with the script ?

**Reply by vpetersson** [reply]
Yes, it's installed in [this playbook](https://github.com/Screenly/Anthias/blob/master/ansible/roles/system/tasks/main.yml).

**Reply by Grougrou4** [reply]
ok thanks

**Comment by Grougrou4** [comment]
to access my pi via a web browser is it simply via his ip or is there a specific port ?

**Reply by vpetersson** [reply]
Yep, just regular http.

**Comment by Grougrou4** [comment]
so i got an error. i try to access local server who require an authentication. is anthias manage it ?

**Reply by vpetersson** [reply]
We would need logs to help but @nicomiguelino should be able to help you out later.

**Reply by Grougrou4** [reply]
how can i use the authentication on web page that requre it ?

**Reply by vpetersson** [reply]
You can't really with Anthias. It is something we support in Screenly tho.
